### DISCUSSION TITLE
关于404 Not found问题的说明

### DISCUSSION THREAD

**Discussion by FranzKafkaYu** [FranzKafkaYu/x-ui#77] [Announcements]
很多朋友在安装/更新版本后进入面板后会提示404 not found 错误。

这是因为自从07.24版本，为了避免面板被扫描爆破，增加了自动生成根路径的功能，只要检测到面板根路径为"/"，则会自动生成新路径并重启面板，此时需要重新更新浏览器内的登录地址，具体内容请参考WIKI文档:
https://github.com/FranzKafkaYu/x-ui/wiki/%E9%9D%A2%E6%9D%BF%E7%99%BB%E5%BD%95

**Comment by iopq** [comment]
that's why I suggest it gets updated when you set the user and password from the command line so people don't run into this issue by accident, and also making sure it's more secure

**Comment by tcsm-cn** [comment]
> 很多朋友在安装/更新版本后进入面板后会提示404 not found 错误。
> 
> 
> 
> 这是因为自从07.24版本，为了避免面板被扫描爆破，增加了自动生成根路径的功能，只要检测到面板根路径为"/"，则会自动生成新路径并重启面板，此时需要重新更新浏览器内的登录地址，具体内容请参考WIKI文档:
> 
> https://github.com/FranzKafkaYu/x-ui/wiki/%E9%9D%A2%E6%9D%BF%E7%99%BB%E5%BD%95
> 
> 

可以关闭这个功能吗？在增加根路径后，我用反向代理就需要手动填写这个目录了

**Comment by uckhanmalek** [comment]
after first day of use, received a 404 error in my panel page.
but my configs is working.
